tests/components/backup/test_websocket.py

```diff
@@ -1637,6 +1637,284 @@ async def test_config_retention_copies_logic(
+                ),
+                "backup-3": MagicMock(
+                    date="2024-11-11T04:45:00+01:00",
+                    with_strategy_settings=True,
+                    spec=ManagerBackup,
+                ),
+                "backup-4": MagicMock(
+                    date="2024-11-12T04:45:00+01:00",
+                    with_strategy_settings=True,
+                    spec=ManagerBackup,
+                ),
+                "backup-5": MagicMock(
+                    date="2024-11-12T04:45:00+01:00",
+                    with_strategy_settings=False,
+                    spec=ManagerBackup,
+                ),
+            },
+            {},
+            {},
+            "2024-11-11T04:45:00+01:00",
+            "2024-11-12T04:45:00+01:00",
+            1,
+            1,
+            2,
+            [call("backup-1"), call("backup-2")],
+        ),
+    ],
+)
+async def test_config_retention_copies_logic_manual_backup(
+    hass: HomeAssistant,
+    hass_ws_client: WebSocketGenerator,
+    freezer: FrozenDateTimeFactory,
+    hass_storage: dict[str, Any],
+    create_backup: AsyncMock,
+    delete_backup: AsyncMock,
+    get_backups: AsyncMock,
+    config_command: dict[str, Any],
+    backup_command: dict[str, Any],
+    backups: dict[str, Any],
+    get_backups_agent_errors: dict[str, Exception],
+    delete_backup_agent_errors: dict[str, Exception],
+    last_backup_time: str,
+    next_time: str,
+    backup_time: str,
+    backup_calls: int,
+    get_backups_calls: int,
+    delete_calls: int,
+    delete_args_list: Any,
+) -> None:
+    """Test config backup retention copies logic."""
+    client = await hass_ws_client(hass)
+    storage_data = {
+        "backups": {},
+        "config": {
+            "create_backup": {
+                "agent_ids": ["test-agent"],
+                "include_addons": ["test-addon"],
+                "include_all_addons": False,
+                "include_database": True,
+                "include_folders": ["media"],
+                "name": "test-name",
+                "password": "test-password",
+            },
+            "retention": {"copies": None, "days": None},
+            "last_attempted_strategy_backup": None,
+            "last_completed_strategy_backup": None,
+            "schedule": {"state": "daily"},
+        },
+    }
+    hass_storage[DOMAIN] = {
+        "data": storage_data,
+        "key": DOMAIN,
+        "version": 1,
+    }
+    get_backups.return_value = (backups, get_backups_agent_errors)
+    delete_backup.return_value = delete_backup_agent_errors
+    await hass.config.async_set_time_zone("Europe/Amsterdam")
+    freezer.move_to("2024-11-11 12:00:00+01:00")
+
+    await setup_backup_integration(hass, remote_agents=["test-agent"])
+    await hass.async_block_till_done()
+
+    await client.send_json_auto_id(config_command)
+    result = await client.receive_json()
+    assert result["success"]
+
+    freezer.move_to(next_time)
```

Comment:
Why do we move to this time in this test?

Reply:
I added it to be able to check the attempted/completed timestamps. But I guess it can be removed since the attempted/completed timestamps in the store are initiated to `None`